src/helpers/TimeExtractor.ts

```diff
@@ -5,7 +5,7 @@ import { parseTimeRange } from "./TimeHelper";
 // Group 1: Time range
 // Group 2: Letter
 // Group 3: Description, optional
-const MATCH_LINE = /(\d{4}-\d{4}):\s?([a-zA-Z])(?:[-\s]+(.+))?/;
+const MATCH_LINE = /(\d{4}\s?-\s?\d{4}):\s?([a-zA-Z])(?:[-\s]+(.+))?/;
```

Comment:
Better to change the regex to `(\d{4})\s?-\s?(\d{4})`, which would make two capture groups, instead of one. This would also require the two next capture groups to be bumped. This would make the other trims redundant